projects/showcase/e2e/src/app.po.ts

```diff
@@ -1,6 +1,6 @@
 import { browser, by, element } from 'protractor';
 import * as path from 'path';
-import { compareScreenshot } from 'snapshot-testing/dist';
+// import { compareScreenshot } from 'snapshot-testing/dist';
```

Comment:
This could be removed

Reply:
I didn't remove it for consistency because the use of compareScreenshot in the same file is also commented time ago with the comment: "// As in Travis and in retina I get different images, for the moment I will not check the snapshots. Try Jest?"  If we want to use it we can uncomment the two parts of code

Reply:
Seems fine for me. Approved